packages/tools/generator-pie-component/src/app/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { startCase, snakeCase } from 'lodash';
+
+function trim (str: string) {
+    return str.replace(/ /g, '');
```

Comment:
Not sure if this is an out-of-the-box function but would it be better to replace any whitespace `/\s/g` rather than just space characters?